apps/web/src/components/PoolCandidatesTable/PoolCandidateFilterDialog.tsx

```diff
@@ -0,0 +1,317 @@
+  requestPolicy: "cache-first", // The list of skills will rarely change, so we override default request policy to avoid unnecessary cache updates.
+};
+
+type PoolCandidateFilterDialogProps = CommonFilterDialogProps<FormValues>;
+
+const PoolCandidateFilterDialog = ({
+  onSubmit,
+  defaultValues,
+}: PoolCandidateFilterDialogProps) => {
+  const intl = useIntl();
+
+  const [{ data, fetching }] = usePoolCandidateFilterDataQuery({ context });
+
+  const classifications = unpackMaybes(data?.classifications);
+  const pools = unpackMaybes(data?.pools);
+  const skills = unpackMaybes(data?.skills);
+
+  const equityOption = (value: string, message: MessageDescriptor) => ({
+    value,
+    label: intl.formatMessage(message),
+  });
+
+  return (
+    <FilterDialog<FormValues> onSubmit={onSubmit} options={{ defaultValues }}>
+      <div
+        data-h2-display="base(grid)"
+        data-h2-gap="base(x1)"
+        data-h2-grid-template-columns="p-tablet(repeat(2, 1fr)) p-tablet(repeat(3, 1fr))"
+      >
+        <div data-h2-grid-column="l-tablet(span 3)">
+          <Combobox
+            id="pools"
+            name="pools"
+            {...{ fetching }}
+            isMulti
+            label={intl.formatMessage(adminMessages.pools)}
+            options={pools.map((pool) => ({
+              value: pool.id,
+              label: getFullPoolTitleLabel(intl, pool),
+            }))}
+          />
+        </div>
+        <Checklist
+          idPrefix="publishingGroups"
+          name="publishingGroups"
+          legend={intl.formatMessage(adminMessages.publishingGroups)}
+          items={enumToOptions(PublishingGroup).map(({ value }) => ({
+            value,
+            label: intl.formatMessage(getPublishingGroup(value)),
+            ariaLabel: intl
+              .formatMessage(getPublishingGroup(value))
+              .replace(
+                intl.locale === "en" ? "IT" : "TI",
+                intl.locale === "en" ? "I T" : "T I",
+              ),
+          }))}
+        />
+        <Checklist
+          idPrefix="equity"
+          name="equity"
+          legend={intl.formatMessage({
+            defaultMessage: "Employment equity",
+            id: "9e6Xph",
+            description: "Label for the employment equity field",
+          })}
+          items={[
+            equityOption("isWoman", getEmploymentEquityGroup("woman")),
+            equityOption(
+              "hasDisability",
+              getEmploymentEquityGroup("disability"),
+            ),
+            equityOption(
+              "isIndigenous",
+              getEmploymentEquityGroup("indigenous"),
+            ),
+            equityOption(
+              "isVisibleMinority",
+              getEmploymentEquityGroup("minority"),
+            ),
+          ]}
+        />
+        <Checklist
```

Comment:
This is probably pre-existing as well so feel free to spin it off.  This seems like a very strange way to query priorities.  Why are we not querying the actual fields in `users` and instead using abstract weight values?

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/64134aa9-e53a-4486-af4f-3040ecdfcd5e)

Reply:
Yeah, this was preexisting. I also found this a little odd.

I'll take a look and see if I can spin this off. I wasn't 100% sure how querying by priority worked so I left it as untouched as I could to be safe 😅

Reply:
Okay, I tried my best at defining the ticket. Hopefully this makes sense, if not adjustments are welcome 👏 

 - https://github.com/GCTC-NTGC/gc-digital-talent/issues/8668

Reply:
Thanks!  I added a bit more detail.